heudiconv/cli/run.py

```diff
@@ -222,6 +253,8 @@ def get_parser():
                             help='Additional queue arguments passed as '
                             'single string of Argument=Value pairs space '
                             'separated.')
+    parser.add_argument('--help-bids', action='store_true', dest='help_bids',
+                        help='Display bids specific help.')
```

Comment:
why not to just add "help" option to `--bids` and react on it?  part of the goal in my suggestion was not to expand the cmdline API options list more ;-)

Reply:
Ah yes that's a good ideal. Will do.

Reply:
Refactored as per @mgxd's suggestion and just included all the help in the argparse help field